.storybook/base.css

```diff
@@ -7,3 +7,8 @@ body {
 table {
     width: 100%;
 }
+
+code.css-1kwwth4 {
```

Comment:
You would want to set this on the styled component rather than the generated css class name.

Reply:
That's a good question. So far in my checks it's always the same name that's generated. It never changes.

I can just have it as a tag only since the other declarations are doing the same in this CSS file. As in `code { ... }`. What do you think?

Reply:
Hi @jbetancur I will change it to just `code` as said. Hope to get this merged because the tables are too wide right now. I'll do this soon today.